builds/jazz-build-module/aws-lambda-events-module.groovy

```diff
@@ -412,7 +416,7 @@ def updateDynamoDbResourceServerless(event_stream_arn){
 
   sh "sed -i -- '/#Start:dynamoDbstreamGetArn/,/#End:dynamoDbstreamGetArn/d' ./policyFile.yml"
   sh "sed -i -- 's|{event_dynamodb_stream_arn}|${event_stream_arn}|g' ./policyFile.yml"
-  sh  "sed -i -- 's/#ResourceDisabled/Resource/g' ./policyFile.yml"
+  sh  "sed -i -- 's/#ResourceDynoDisabled/Resource/g' ./policyFile.yml"
```

Comment:
ResourceDynamoDb?

Reply:
Renamed to ResourceDynamoDbDisabled

Reply:
Its already updated. 
https://github.com/tmobile/jazz/pull/690/files#diff-12807d9a54f73648dc7b6edcbc809d1aR72